web/src/components/CollapsibleList.js

```diff
@@ -44,6 +44,17 @@ class ListItem extends Component {
             </button>
           )}
         </div>
+        <div className="mb1 h5 flex justify-between visibility--print">
+          <button
+            type="button"
+            className="btn btn-no-focus p1 flex-auto left-align bg-blue-light rounded"
+            onClick={this.toggle}
+          >
+            <header className="flex items-center justify-between">
+              {header}
+            </header>
+          </button>
+        </div>
```

Comment:
This is for the little `x` button on the far right side of the mini-collapsible list headers, which will delete an item:

![Screen Shot 2019-03-13 at 10 29 29 AM](https://user-images.githubusercontent.com/1775733/54291718-fd545580-457a-11e9-9810-77e36bf6304d.png)

Whether to display the `x` is conditioned on whether or not there's a delete handler - no handler, no `x` button.  Because the `x` button provides the rounding on the right side, just hiding the `x` button messes up the rounding.  Instead of doing lots of extra logic work here, I just split this into screen and print versions, where the print version doesn't do any of the logic because the `x` is always omitted.

Reply:
Wild idea: what if we blew away the button altogether and just displayed the text??

Reply:
You mean in print?  That...  makes too much sense.  I was hesitant to change too much other than just making the clickable buttons go away, but replacing the button with just a header with text in it makes sense.  Happy to do that if you think it's a better approach!

Alternatively, these headers are useless in print because they get truncated (as in the screenshot) and we need a totally different view for them in print?  I think that would be a separate PR, but maybe worth asking @jenniferthibault what she thinks while we still have the chance?

Reply:
I think that when the designs from https://github.com/18F/cms-hitech-apd/issues/1330 are implemented, we won't have this problem because we'll be using the Review component, which neither truncates text nor has little x buttons. So do whatever you gotta do to get there :)